From f50f74f2bba6aca59e2688a4b5229c27aad50c28 Mon Sep 17 00:00:00 2001
From: Angelo Compagnucci <angelo@amarulasolutions.com>
Date: Mon, 11 Nov 2019 16:27:10 +0100
Subject: [PATCH] core/execution: use USE_BACKWARD_CPP only when HAVE_BACKTRACE

If any backtrace support is available, like on uclibc, USE_BACKWARD_CPP
is not available either, thus disabling it.

Signed-off-by: Angelo Compagnucci <angelo@amarulasolutions.com>
---
 core/execution.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/execution.cpp b/core/execution.cpp
index 6e7c6eaee..9551a2887 100644
--- a/core/execution.cpp
+++ b/core/execution.cpp
@@ -31,7 +31,7 @@
 
 #include <QtGlobal>
 
-#if defined(Q_OS_LINUX) && !defined(Q_OS_ANDROID)
+#if defined(Q_OS_LINUX) && !defined(Q_OS_ANDROID) && defined(HAVE_BACKTRACE)
 #include <backward.hpp>
 #define USE_BACKWARD_CPP
 #else
